src/main/features/offer/views/declaration.njk

```diff
@@ -48,12 +39,9 @@
         </ul>
 
         {% if not isThroughAdmissions %}
-          <p>
-            {{ t('To apply, download and complete form N244 and send it to the County Court Business Centre. You need to say on the form which of the 3 options you’re applying for.', {
+          <p>{{ t('To apply, download and complete form N244 and send it to the County Court Business Centre. You need to say on the form which of the 3 options you’re applying for.', {
               n244formUri: 'http://formfinder.hmctsformfinder.justice.gov.uk/n244-eng.pdf',
-              countyCourtBusinessCentreUri: 'https://courttribunalfinder.service.gov.uk/courts/county-court-business-centre-ccbc'
-            }) | safe }}
-          </p>
+              countyCourtBusinessCentreUri: 'https://courttribunalfinder.service.gov.uk/courts/county-court-business-centre-ccbc' }) | safe }}</p>
```

Comment:
These aren't injected anywhere in the string, or am I missing something?

Reply:
good spot. I was just cleaning up the whitespace around this. shall put them in

Reply:
actually no. I'll get the testers to review that page as a separate issue. Don't want to confuse these PRs and there may be other things wrong on that page.